Describe local installation with CLI (#32)
capactio/website: docs/installation/local.md

## Create local Kubernetes cluster

To create local [kind](https://kind.sigs.k8s.io/) cluster, run the following command:

```bash
capact environment create kind --wait 5m
```

> **NOTE** You can customize the behavior with additional flags. See the all options in the [`capact environment create kind`](../cli/commands/capact_environment_create_kind.md) command documentation.

## Install Capact

Once you have your local Kubernetes cluster set up, install Capact with the following command:

```bash
capact install
```

> **NOTE** You can customize the behavior with additional flags. See the all options in the [`capact install`](../cli/commands/capact_install.md) command documentation.

Wait for the command to finish.

## Next steps

Configure Capact CLI to connect to your fresh local Capact installation. Follow the steps in the [First use](../cli/getting-started#first-use) section of the CLI Getting started guide. Enjoy using Capact!

## Cleanup

To remove your local Kubernetes cluster created in the [Create local Kubernetes cluster](#create-local-kubernetes-cluster) section, run the following command:

```bash
capact environment delete kind
```

> > **NOTE:** This command deletes the local cluster with all data, including your run ran Actions history and Local Hub with stored TypeInstances. Make sure you cleaned up all external resources (e.g. managed PostgreSQL databases) based on the TypeInstance data.

Reviewer: ```suggestion
> **NOTE:** This command deletes the local cluster with all data, including your ran Actions history and Local Hub with stored TypeInstances. Make sure you cleaned up all external resources (e.g. managed PostgreSQL databases) based on the TypeInstance data.
```

# Local Installation

Use Capact CLI to create local Kubernetes cluster and install Capact on it.

## Prerequisites
- [Docker](https://www.docker.com/) installed

> - [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed

Reviewer: can be removed after applying my suggestion

- Capact CLI installed according to the [Getting Started](./../cli/getting-started.mdx#install) guide

## Create local Kubernetes cluster

To create local [kind](https://kind.sigs.k8s.io/) cluster, run the following command:

```bash

> capact environment create kind

Reviewer: ```suggestion
capact environment create kind --wait 5m
```
Previously we had 5min, IMO it's max that it may take. Normally, it's under 1min.

```

Verify that your local cluster is running by listing all Nodes:

```bash
kubectl get nodes
```

The output should be similar to:

```bash
NAME                            STATUS   ROLES    AGE    VERSION
kind-dev-capact-control-plane   Ready    master   3m5s   v1.19.1

> ```

Reviewer: TBH I would prefer to skip that. This doesn't show how to debug or fix the problem if the node is not ready. Such information user will get after executing first command.

## Install Capact

> Once you have your local Kubernetes cluster set up, install Capact with the following command:

Reviewer: ```suggestion
Once you have your local Kubernetes cluster setup, install Capact with the following command:
```

Author: > Setup is a noun that refers to an arrangement. Set up is a phrasal verb that refers to making preparations for something.
In this context it's a verb, so I will keep it as it is.
